catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -114,7 +114,7 @@
 
   public static final String POINT_KEY = "pt";
 
-  public static final int GET_BY_ID_LIMIT = 200;
+  public static final int GET_BY_ID_LIMIT = 100;
```

Comment:
What is it about CSW that results in the url being too long?

Reply:
@millerw8 when you request a large amount of deletions DDF will batch them into partitions of IDs to send to solr. Each partition gets strung into a url that is then sent to solr, and it's currently too long at `200` and so the http request gets a 400 response from Solr

Reply:
This isn't limited by CSW, as it's in the SolrMetacardClientImpl so it would affect any large delete requests sent to Solr.

Reply:
The 200 limit was tested and worked before.  I wonder if a Jetty upgrade changed the URL length constraint back to the default.